gallery/src/ha-gallery.js

```diff
@@ -151,6 +188,14 @@ class HaGallery extends PolymerElement {
   _backTapped() {
     document.location.hash = '';
   }
+
+  _computeLovelace(dems) {
```

Comment:
demos

Reply:
That was to avoid shadowing the `demos` var at the root level of the module. I have renamed that one now to `DEMOS`